match.py

```diff
@@ -0,0 +1,5 @@
+import requests 
+
+r = requests.get("https://github.com", verify=False)
```

Comment:
[Disabled certificate validation: DS130821](https://github.com/microsoft/devskim/guidance/DS114352.md)
Extend default certificate validation, but do not disable or override default rules.

*Replacement:* Always use a valid certificate, even during testing.